### PR TITLE
ci(benchmarking): Added benchmarking stage

### PR DESCRIPTION
**Pull request by msohailhussain**
Summary
-------

-  The “what”; a concise description of each logical change
-  Another change

The “why”, or other context.

Test plan
---------

Issues
------

-  “THING-1234” or “Fixes #123”

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -30,17 +31,23 @@ jobs:
...
+        - mkdir $HOME/travisci-tools && pushd $HOME/travisci-tools && git init && git pull https://$CI_USER_TOKEN@github.com/optimizely/travisci-tools.git sohail/refactorscript && popd
       script:
-        - "$HOME/travisci-tools/fsc-trigger/trigger_fullstack-sdk-compat.sh"
+        - "$HOME/travisci-tools/trigger-script-with-status-update.sh"
       after_success: travis_terminate 0
+    - <<: *integrationtest
+      stage: 'Benchmarking Tests'
+      env: SDK=python REPO_SLUG=Benchmarking
+      script:
+        - "$HOME/travisci-tools/trigger-script-with-status-update.sh yasir/bench-exp-support"
```
Comment (juancarlostong) [inline .travis.yml]: the yasir/bench-exp-support is for testing right

File: .travis.yml [hunk 2]
```diff
@@ -17,7 +17,8 @@ after_success:
 # Linting and Integration tests need to run first to reset the PR build status to pending.
 stages:
   - 'Linting'
-  - 'Integration tests'
+  - 'Integration Tests'
```
Comment (juancarlostong) [inline .travis.yml]: travis gui will lowercase this for you even though you specify upppercase... it's annoying :(

File: .travis.yml (hunk 1 above)
Comment (msohailhussain) [reply .travis.yml]: yup

File: .travis.yml (hunk 2 above)
Comment (msohailhussain) [reply .travis.yml]: yeah realized, will change it.

File: .travis.yml (hunk 1 above)
Comment (mikeng13) [reply .travis.yml]: We can remove the custom branch and merge in now right?

Comment (msohailhussain) [reply .travis.yml]: yes, about to remove.

**Pull request merged**